cmd/osbuild-dnf-json-tests/main_test.go

```diff
@@ -50,38 +52,35 @@ func TestCrossArchDepsolve(t *testing.T) {
 	rpm := rpmmd.NewRPMMD(dir)
 
 	// Load repositories from the definition we provide in the RPM package
-	repositories := "/usr/share/osbuild-composer"
+	repoDir := "/usr/share/osbuild-composer"
```

Comment:
Are these tests always run from a built and installed package? If they run from the git checkout there is a possibility of not using the correct set of repos.

Reply:
Yes, they are meant to be run as integration tests. It has the `integration` build constraint at the top of the file.

I guess we could run it from source, but we'd need to change this path and set `PATH` to find the right `dnf-json`. I don't think it's worth making this test more complex in such a way.